Makefile.include

```diff
@@ -317,16 +317,16 @@ distclean:
 	-@rm -rf $(BINDIRBASE)
 
 flash: all
-	$(AD)command -v $(FLASHER) >/dev/null 2>&1 || \
+	@$(AD)command -v $(FLASHER) >/dev/null 2>&1 || \
```

Comment:
I think `@$(AD)` doesn't make much sense, if this output should always be prevented, just use `@command`. (`$(AD)` is either set to `"@"` (if `QUIET == 1`) or unset.)